tests/testthat/test-grid_search.R

```diff
@@ -46,40 +46,63 @@ test_that("Small grid search works with new model", {
   set.seed(1)
 
   data <- readRDS("hospital_model_data.rds")
- 
+
   # Parameters for run
   min_beta <- 0.1
   max_beta <- 0.2
   beta_step <- 0.05
   first_start_date <- "2020-01-21"
   last_start_date <- "2020-01-22"
   day_step <- 1
-  
+
   scan_results = scan_beta_date(
     min_beta = min_beta,
     max_beta = max_beta,
     beta_step = beta_step,
-    first_start_date = first_start_date, 
-    last_start_date = last_start_date, 
+    first_start_date = first_start_date,
+    last_start_date = last_start_date,
+    day_step = day_step,
+    data = data,
+    sircovid_model = hospital_model(),
+    scale_prior = 0.003541667,
+    shape_prior = 36
+    )
+
+
+  expect_warning(
+    scan_beta_date(
+    min_beta = min_beta,
+    max_beta = max_beta,
+    beta_step = beta_step,
+    first_start_date = first_start_date,
+    last_start_date = last_start_date,
     day_step = day_step,
     data = data,
     sircovid_model = hospital_model(),
     scale_prior = 0.003541667,
-    shape_prior = 36)
+    shape_prior = 36,
+    ## 292 0s before a non-zero number when I run stuff.
```

Comment:
A better test might instead be to move the search boundaries so they are cutting through the probability, as this could be getting pretty close to machine precision. So perhaps try a min_beta of 0.17 (or look at the plot output from this to see where you could put a plot boundary to cut through the main mass)